compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -767,6 +770,10 @@ class Definitions {
     lazy val RuntimeTuple_concat: Symbol = RuntimeTupleModule.requiredMethod("concat")
     lazy val RuntimeTuple_toArray: Symbol = RuntimeTupleModule.requiredMethod("toArray")
     lazy val RuntimeTuple_productToArray: Symbol = RuntimeTupleModule.requiredMethod("productToArray")
+    lazy val RuntimeTuple_isInstanceOfTuple: Symbol = RuntimeTupleModule.requiredMethod("isInstanceOfTuple")
+    lazy val RuntimeTuple_isInstanceOfEmptyTuple: Symbol = RuntimeTupleModule.requiredMethod("isInstanceOfEmptyTuple")
+    lazy val RuntimeTuple_isInstanceOfNonEmptyTuple: Symbol = RuntimeTupleModule.requiredMethod("isInstanceOfNonEmptyTuple")
+    lazy val RuntimeTuple_isInstanceOfPair: Symbol = RuntimeTupleModule.requiredMethod("isInstanceOfPair")
```

Comment:
I think we can just keep one of isInstanceOfEmptyTuple and isInstanceOfPair, since they are the same. My vote would be to keep isInstanceOfEmptyTuple and to drop isInstanceOfPair.